warehouse/ingest-json/src/test/resources/config/ingest/tvmaze-edge-definitions.xml

```diff
@@ -34,180 +34,16 @@ http://www.springframework.org/schema/util/spring-util-4.0.xsd">
                                in the graph.
```

Comment:
This comment above is no longer valid, right?

Reply:
ah, yes, I should update that